great-idea/services.html

```diff
@@ -2,105 +2,125 @@
 
 <html lang="en">
 <head>
-  <meta charset="utf-8">
+<meta charset="utf-8">
 
-  <title>Great Idea!</title>
+<title>Great Idea!</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+<link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
+<link rel="stylesheet" href="css/services.css">
 
-  <!--[if lt IE 9]>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
-  <![endif]-->
+<!--[if lt IE 9]>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+<![endif]-->
 </head>
 
-<body>
-
-
-Services
-Product
-Vision
-Features
-About
-Contact
-
-<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-
+<body class="main-container">
+<header>
+  <nav>
+    <a>Services</a>
+    <a>Product</a>
+    <a>Vision</a>
+    <a>Features</a>
+    <a>About</a>
+    <a>Contact</a>
+  </nav>
+  <div>
```

Comment:
This outer div might be unnecessary as the a tag itself is an element you can flex.